web-animations-1/Overview.bs

```diff
@@ -3807,9 +3807,9 @@ if <em>all</em> of the following conditions are true:
     [=removed replace state|removed=].
 *   The [=animation=] is associated with a [=monotonically increasing=]
     [=timeline=].
-*   The [=animation=] has an associated [=target effect=].
-*   The [=target effect=] associated with the [=animation=] is [=in effect=].
-*   The [=target effect=] has an associated [=effect target=].
+*   The [=animation=] has an [=associated effect=].
+*   The [=animation=]'s [=associated effect=] is [=in effect=].
+*   The [=associated effect=] has an [=effect target=].
```

Comment:
(Nit: I guess for consistency we should make the last bullet point also say `The [=animation=]'s [=associated effect=]...`.)